src/resources/i18n/emailpro/Messages_fr_FR.xml

```diff
@@ -63,6 +63,8 @@
    <translation id="emailpro_tab_domain_modify_domain_tooltip" qtlid="447950">L'antispam Vade Secure est compatible avec le mode non-autoritatif (uniquement si vos comptes pop/imap sont hébergés par OVH). Dans ce cas, il est conseillé d'utiliser : <strong>{0}</strong>, <strong>{1}</strong>, <strong>{2}</strong> et <strong>{3}</strong><br/><br/> Si vous utilisez le mode non-autoritatif avec des comptes e-mail non hébergés par OVH, vous devez utiliser : <strong>{4}</strong> (voir section domaines associés)</translation>
    <translation id="emailpro_tab_domain_manage" qtlid="404395">: gérer</translation>
 
+   <translation id="emailpro_configuration_action_title">Gérer la politique de sécurité</translation>
```

Comment:
Aren't there any more translation whose id contains the word "exchange" ?

Reply:
No, it seems to be the only one left